src/Uno.UI/Extensions/UIViewExtensions.iOSmacOS.cs

```diff
@@ -206,6 +206,25 @@ public static IEnumerable<_View> FindSubviews(this _View view, Func<_View, bool>
 			}
 		}
 
+		public static IEnumerable<_View> FindSubviewsReverse(this _View view, Func<_View, bool> selector, int maxDepth = 20)
+		{
+			for (int i = view.Subviews.Length - 1; i >= 0; i--)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'view' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5459832)